lib/vpt/bumper/bumper-animation.ts

```diff
@@ -44,15 +44,15 @@ export class BumperAnimation implements IAnimation {
 		this.state = state;
 	}
 
-	public init(physics: PlayerPhysics): void {
-		this.timeMsec = physics.timeMsec;
+	public init(timeMsec: number): void {
+		this.timeMsec = timeMsec;
 	}
 
-	public updateAnimation(physics: PlayerPhysics, table: Table): void {
+	public updateAnimation(timeMsec: number, table: Table): void {
 
-		const oldTimeMsec = this.timeMsec < physics.timeMsec ? this.timeMsec : physics.timeMsec;
-		this.timeMsec = physics.timeMsec;
-		const diffTimeMsec = (physics.timeMsec - oldTimeMsec);
+		const oldTimeMsec = this.timeMsec < timeMsec ? this.timeMsec : timeMsec;
```

Comment:
minor: a bit hard to read, what the difference between this.timeMsec and timeMsec.. where are they used?

Reply:
Yeah, agree... `this.timeMsec` is the "animation's time", `timeMsec` is what's passed to the function, so "to which time to animate to". Mabye `newTimeMsec` for the new time?